docs/storybook/pages/wall.stories.tsx

```
@@ -0,0 +1,389 @@
+                </Text>
+              </Column>
+            </Columns>
+            <Breadcrumb>
+              <BreadcrumbItem href={'#'}>Accueil</BreadcrumbItem>
+              <BreadcrumbItem href={'#'} active>
+                téléphone mobile
+              </BreadcrumbItem>
+            </Breadcrumb>
+            <Accordion children>
+              <AccordionItem children>
+                <AccordionHeader>Mentions légales</AccordionHeader>
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)